app/src/forms/admin/index.js

```diff
@@ -2,6 +2,14 @@ const dataErrors = require('../common/middleware').dataErrors;
 const routes = require('./routes');
 const setupMount = require('../common/utils').setupMount;
 
+const swaggerSpec = require('../../swagger/swaggerui');
+
+swaggerSpec.tags.push({
+  name: 'Admin',
+  description:
+    'These API endpoints handle the input data provided by a user that completes the form, for example, writing the form data to the database, or exporting form submissions created during a defined time period.',
+});
```

Comment:
This is the description of the `submissions` endpoint in the previous docs. Do we want it to be what the `admin` endpoint used to be?

> These API endpoints are used for elevated permission administrative calls. All calls are secured by a role that operational team members will be granted and allow fetching some details about forms without needing a user->form permission.

Reply:
Fix